src/decode.rs

```diff
@@ -3947,10 +3946,10 @@ unsafe fn setup_tile(
 
 unsafe fn read_restoration_info(
     t: &mut Rav1dTaskContext,
-    f: &Rav1dFrameData,
     lr: &mut Av1RestorationUnit,
     p: usize,
     frame_type: Rav1dRestorationType,
+    debug_block_info: bool,
```

Comment:
Since we precompute `debug_block_info`, we need to make sure `fn read_restoration_info` doesn't modify `debug_block_info!(f, t)`.  This is true here, but I think we should make it obvious by not passing `t: &mut Rav1dTaskContext`, but just `ts: &mut Rav1dTileState`.  No other fields of `t` are used.  That'll also probably help a little bit with borrowck issues later.

Reply:
Good idea, done!